awq/entry.py

```diff
@@ -114,8 +123,8 @@ def skip(*args, **kwargs):
             exit(0)
         else:
             # Inference with fake quant
-            # Init model on GPUs:
-            kwargs = {"device_map": "balanced", "torch_dtype": torch.float16}
+            # Init model on CPU:
+            kwargs = {"torch_dtype": torch.float16, "low_cpu_mem_usage": True}
```

Comment:
`low_cpu_mem_usage` is needed here as loading HF models on CPU with fp16 dtype is slow, more details here: https://huggingface.co/mosaicml/mpt-7b-instruct/discussions/6#6470dec93df93fddece5fcde and